src/main/java/com/softserve/teamproject/service/TeacherGroupsManipulationService.java

```diff
@@ -4,12 +4,17 @@
 import java.util.List;
 import org.springframework.security.access.prepost.PreAuthorize;
 
+/**
+ * Interface that provides the methods needed to specify the actions that a teacher can perform with
+ * the groups.
+ */
 public interface TeacherGroupsManipulationService {
 
   /**
    * The access to this method is secured and accessible only to the users with the "teacher" role.
+   *
    * @param teachersName String value, name of the teacher
-   * @return List<Group>: </>List of Groups of this authenticated teacher
+   * @return List<Group>: List of Groups of this authenticated teacher
    */
   @PreAuthorize("hasAuthority('teacher')")
```

Comment:
@PreAuthorize("hasAuthority('teacher')") Story 6  => User with category role "It Academy" should have access to his groups. "It Academy" includes teacher, coordinator and admin.